LICENSE-MIT

```diff
@@ -1,3 +1,5 @@
+The MIT License (MIT)
```

Comment:
Just double-checking this is ok with people who understand legal stuff

Reply:
I don't understand legal stuff but i think this is not needed:

https://opensource.org/license/mit/
https://en.wikipedia.org/wiki/MIT_License